### DISCUSSION TITLE
Feature: Support embed media via Hook link

### DISCUSSION THREAD

**Discussion by wenlzhang** [aidenlx/media-extended#651] [Ideas]
Is it possible to support embed media files in the form of Hook links? The format might be something like: `![Audio.mp3](hook://file/xxx.mp3)`.

Here is something about Hook link:
[Contacting Developers of Other Apps and Information for Developers – Hook](https://hookproductivity.com/help/integration/other-app-developers)

**Comment by ChristinWhite** [comment]
This would be fantastic, it would let me get videos out of my vault and into a better location without needing to worry about paths ever changing.
